Gododir/gen.go

```diff
@@ -216,24 +239,36 @@ var (
 			},
 		},
 	}
+	failLock = &sync.Mutex{}
+	fails    []string
```

Comment:
This is mostly academic in this case, but seeing that these two objects are tightly coupled I would put them in a struct and provide methods for getting/setting so that any invariants can be transparently maintained.

Reply:
Even better, then you wouldn't need to name the Mutex. You'd be able to Lock/Unlock directly off of the object.

Reply:
Fixed